### PR TITLE
f2fs-tools: update to 1.15.0

### PR DESCRIPTION
**Pull request by PolynomialDivision**

### CODE REVIEW

**Comment by PolynomialDivision** [comment]
I guess it has something to do with:
https://git.kernel.org/pub/scm/linux/kernel/git/jaegeuk/f2fs-tools.git/commit/configure.ac?id=c48335416a09b881d2309b8f88c9b3d1441f4a9c

File: package/utils/f2fs-tools/patches/200-resize_f2fs-fix_wrong_ovp_calculation.patch [hunk 1]
```diff
@@ -34,9 +34,9 @@ Signed-off-by: Jaegeuk Kim <jaegeuk@kernel.org>
...
 --- a/lib/libf2fs_zoned.c
 +++ b/lib/libf2fs_zoned.c
-@@ -495,6 +495,9 @@ uint32_t f2fs_get_usable_segments(struct
- 	int i, j;
- 	uint32_t usable_segs = 0, zone_segs;
+@@ -508,6 +508,9 @@ uint32_t f2fs_get_usable_segments(struct
+ 	if (c.func == RESIZE)
+ 		return get_sb(segment_count_main);
  
 +	if (c.func == RESIZE)
```
Comment (hauke) [inline package/utils/f2fs-tools/patches/200-resize_f2fs-fix_wrong_ovp_calculation.patch:41]: This patch was included upstream, you can remove it.

Comment (PolynomialDivision) [reply package/utils/f2fs-tools/patches/200-resize_f2fs-fix_wrong_ovp_calculation.patch]: Thanks for spotting this. The patch was still able to apply correctly.

Comment (hauke) [reply package/utils/f2fs-tools/patches/200-resize_f2fs-fix_wrong_ovp_calculation.patch]: This sometimes happens with small patches, normally I have a second look at patches were much changes in the refresh.

**Comment by hauke** [comment]
Thank you for your patch, I applied it to master in 8b9e806160118eb115ea26a1fbf381801185bdb6.

**Pull request closed without merging**